### PR TITLE
Adding RL Task

### PR DESCRIPTION
**Pull request by simoninithomas**
Okay so I've just added RL task:
- I've modified Tasks.ts
- I've added the icon to js/src/lib/components/icons
- I've modified ModelPipelineIcon.svelte

I might have missing something though?

Thanks for the feedback

### CODE REVIEW

File: js/src/lib/interfaces/Types.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ export enum PipelineType {
 	"image-to-text"                                           = "Image-to-Text",
 	/// others
 	"structured-data-classification"                          = "Structured Data Classification",
+	"reinforcement-learning"				  = "Reinforcement Learning",
```
Comment (Pierrci) [inline js/src/lib/interfaces/Types.ts:41]: can you use spaces instead of tabs? (same elsewhere)

**Review by Pierrci — COMMENTED** [review]
> I think the style error comes from the Icon since the HTML is in "compact mode" compared to the others icons?

You need to run prettier w/ `npm run format`, this should fix things

File: js/src/lib/interfaces/Types.ts (hunk 1 above)
Comment (simoninithomas) [reply js/src/lib/interfaces/Types.ts:41]: Sure!

Comment (julien-c) [reply js/src/lib/interfaces/Types.ts:41]: BTW this shouldn't take engineering time from @Pierrci IMO, you should check with library maintainers @LysandreJik @osanseviero @muellerzr when CI is red on your PR

**Pull request closed without merging**